src/persmission-handler.js

```diff
@@ -0,0 +1,126 @@
+/* global localStorage */
+const yo = require('yo-yo')
+const csjs = require('csjs-inject')
+const modalDialog = require('./app/ui/modaldialog')
+
+const css = csjs`
+.permission p {
+  text-align: center;
+}
+.images {
+  display: flex;
+  justify-content: center;
+  align-item: center;  
+}
+.images img {
+  width: 40px;
+  height: 40px;
+}
+`
+
+export class PermissionHandler {
```

Comment:
should that inherit the base class?

Reply:
There is no base class. It's just an example in `remix-plugin`.